visualizations/frontend/filters/v2/TRangeInput.vue

```diff
@@ -0,0 +1,108 @@
+<template>
```

Comment:
Maybe we should call it `TRangeSlider`? Just to match the naming inside the file. Also, does Ezra have a preference on what the component should be called?

Reply:
Yes, there is already a old and broken component called that, for now I chose a different name so that it will be easy to review, once this PR gets approved I'll rename it to `TRangeSlider`

Reply:
oh I see. Is the rollout plan to Implement these components, test them and then remove the old components?

Reply:
Absolutely